cmd/root.go

```diff
@@ -157,10 +158,13 @@ func hpcToolkitRepo() (repo *git.Repository, dir string, err error) {
 	dir = filepath.Dir(e)
 
 	repo, err = git.PlainOpen(dir)
-	if err == nil && isHpcToolkitRepo(*repo) {
-		return repo, dir, err
+	if err != nil {
+		return
+	}
+	if isHpcToolkitRepo(*repo) {
+		return
 	}
-	return nil, "", err
+	return nil, "", errors.New("Not the hpc-toolkit repo")
```

Comment:
This message may not be very clear to an user. Do we not expect it to be ever reached? otherwise, could we try to have something more clear, like, `Unable to find the hpc-toolkit git repo in the file system.` or something of the sort.

Reply:
Addressed